<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-534BRGJ" ignored="2" name="Test" tests="5" failures="1" timestamp="22 Nov 2019 21:23:50 GMT" time="118.987" errors="0">
  <testcase name="login" time="9.118" classname="com.learnautomation.testCases.LoginTest">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="34.148" classname="com.learnautomation.testCases.LoginTest">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="34.034" classname="com.learnautomation.testCases.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.learnautomation.testCases.LoginTest.login(LoginTest.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="loginApp" time="8.55" classname="com.learnautomation.testCases.LoginLogoutTest"/>
  <testcase name="logoutApp" time="3.243" classname="com.learnautomation.testCases.LoginLogoutTest"/>
  <testcase name="@BeforeSuite setUpSuite" time="0.0" classname="com.learnautomation.pages.BaseCalss">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUpSuite -->
  <testcase name="@AfterSuite tearDownSuite" time="0.0" classname="com.learnautomation.pages.BaseCalss">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownSuite -->
</testsuite> <!-- Test -->
